src/gtktsm/gtktsm-win.c

```diff
@@ -84,8 +93,15 @@ static void gtktsm_win_init(GtkTsmWin *win)
 	gtk_container_add(GTK_CONTAINER(win), GTK_WIDGET(p->term));
 	gtk_widget_show(GTK_WIDGET(p->term));
 
+    g_signal_connect(G_OBJECT(p->term),
+			 "terminal-title-changed",
+
+			 G_CALLBACK(win_title_fn),
+			 win);
+
 	g_signal_connect(G_OBJECT(p->term),
 			 "terminal-stopped",
+
```

Comment:
Haha, that was a holdover from a painful point where I realized my editor was indenting using spaces instead of tabs, and I had to change them all over.